scripts/west_commands/runners/openocd.py

```diff
@@ -109,13 +114,19 @@ def do_flash(self, **kwargs):
         if self.pre_cmd is not None:
             pre_cmd = ['-c', self.pre_cmd]
 
+        pre_init_cmd = []
+        if self.pre_init_cmd is not None:
+            pre_init_cmd = ['-c', self.pre_init_cmd]
+
         post_cmd = []
         if self.post_cmd is not None:
             post_cmd = ['-c', self.post_cmd]
 
         self.logger.info('Flashing file: {}'.format(self.hex_name))
         cmd = (self.openocd_cmd +
-               ['-f', self.openocd_config,
+               ['-f', self.openocd_config] +
+                pre_init_cmd +
+               [
```

Comment:
Just a whitespace nit -- can you join this with the next line so it's

```
['-c', 'init',
```

?

Otherwise this looks fine to me and I'm +1 if it helps you and @erwango